composer.json

```diff
@@ -22,17 +22,23 @@
         "issues": "http://github.com/liip/RMT/issues"
     },
     "require": {
-        "php": ">=5.3.9",
+        "php": "^5.6|^7.0",
         "symfony/console": "~2.3|^3.0",
         "symfony/yaml": "~2.3|^3.0",
         "symfony/process": "~2.3|^3.0",
         "vierbergenlars/php-semver": "~3.0",
         "sensiolabs/security-checker": "~2.0|^3.0|^4.0"
     },
+    "require-dev": {
+        "phpunit/phpunit": "^5.6.8|^6.0"
```

Comment:
@acrobat Have a look at https://github.com/sebastianbergmann/phpunit/blob/master/src/Framework/Assert.php#L531. Now all assertions are static called. It was a complete rebuild of the assertions. So i suggest to provide only support for Unit Tests depending on Version < 6.
Otherwise we will need to update all test files to use the statics and this will drop the support for PHP 5.6 (https://github.com/sebastianbergmann/phpunit/wiki/Release-Announcement-for-PHPUnit-6.0.0#dropping-support-for-php-56).